### DISCUSSION TITLE
Add Files to the dock as default

### DISCUSSION THREAD

**Discussion by mpanhans** [elementary/default-settings#269] [Ideas]
I know there has been a previously [closed issue](https://github.com/elementary/default-settings/issues/191) about this, so rather than open a new issue in this repo I thought to start a discussion.

There are three simple reasons the Files app should be in the dock by default:
1. New users do not know what the file browser on the operating system is called. Putting Files in the dock by default is the best way to introduce it to them.
2. Some users will want Files in the dock and some will not. But there is an asymmetric friction between removing it for those who don't want it (simply unselect "keep in dock"), compared to adding it especially if you do not know or forget the name of the file browser app in elementary.
3. Guest sessions. When elementary is deployed in settings where some people will be using Guest Sessions (such as a public computer lab, community center, etc), it will make sense to have straightforward access to the file system, for those users to say access downloaded files or files on a USB.

In addition, there is also a deeper reason to include the Files app in the dock. I understand the reasoning for the decision made in the past, that "we encourage an app-based workflow over a file-based one." But I don't think that decision reflects the full picture. Specifically, I think a good OS should teach the user intuitively how and where it stores files. 

A recent article in The Verge entitled ["File Not Found"](https://www.theverge.com/22684730/students-file-folder-directory-structure-education-gen-z) talks about how the generation that grew up with iPhones and Google Docs do not even understand how files are stored on a computer:

> Garland thought it would be an easy fix. She asked each student where they’d saved their project. Could they be on the desktop? Perhaps in the shared drive? But over and over, she was met with confusion. “What are you talking about?” multiple students inquired. Not only did they not know where their files were saved — they didn’t understand the question.
> 
> Gradually, Garland came to the same realization that many of her fellow educators have reached in the past four years: the concept of file folders and directories, essential to previous generations’ understanding of computers, is gibberish to many modern students.

In STEM and data science fields, knowing file structure is key for work and research (at least up to now). Some professors have started doing a two hour lecture just on file structures for their students. It's possible using file directories is the way of the past, and some of us are just clinging to a dying way of digital organization. But at least I hope the devs think carefully about what future they want to usher in, and the role they are playing in the education of the next generation of computer users.

**Comment by danirabbit** [comment]
I reopened #191 and left a comment there. In the future, please feel free to continue to comment on a closed issue report rather than create a separate discussion :)
